test/test_mediafile.py

```diff
@@ -373,7 +350,6 @@ class ReadWriteTestBase(ArtTestMixin, GenreListTestMixin,
         'genre',
         'lyricist',
         'composer',
-        'composer_sort',
```

Comment:
This list *does* have an `artist_sort`, so the `composer_sort` is probably also applicable here.